src/validators/validators.service.ts

```diff
@@ -82,7 +82,6 @@ export class ValidatorsService {
 
   public async getMetaDataFromStorage(): Promise<ConsensusMeta | null> {
     if (this.isDisabledRegistry()) {
-      this.logger.warn('ValidatorsRegistry is disabled in API');
```

Comment:
maybe we should keep the log to make it easier to see when working with clients? just change it from warn to log/info.

Reply:
This log is unnecessary; we only need to log it once during job initialization. Currently, it writes to the log every time someone requests /status, which creates a false impression and clutters the readability of the logs